memflow-win32/src/kernel/start_block/aarch64.rs

```diff
@@ -0,0 +1,50 @@
+use crate::error::{Error, Result};
+use crate::kernel::StartBlock;
+
+use std::convert::TryInto;
+
+use memflow::architecture::arm::aarch64;
+use memflow::types::{size, Address};
+
+pub const PHYS_BASE: Address = Address::from_u64(0x40000000);
+
+fn find_pt(addr: Address, mem: &[u8]) -> Option<Address> {
```

Comment:
A doc comment telling the user about the mem size requirement would be helpful here (just to make it extra clean).